lib/analytics/lib/src/analytics/backend/logging_analytics_backend.dart

```diff
@@ -6,32 +6,35 @@
 //
 // SPDX-License-Identifier: EUPL-1.2
 
+import 'dart:developer' as developer;
+
 import '../analytics.dart';
 
 class LoggingAnalyticsBackend extends AnalyticsBackend {
   @override
   void log(String name, [Map<String, dynamic>? data]) {
-    print("AnalyticsEvent $name received. Data: $data");
+    developer.log("AnalyticsEvent $name received. Data: $data");
```

Comment:
I needed to use a prefix because otherwise we would call our analytics `log` method